recipes/reprounzip-qt/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "reprounzip-qt" %}
+{% set version = "1.0.11" %}
+{% set sha256 = "5cb71f598753150249674208ca47316fe660bdb3feafeaab039874af104f64c4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - pyqt <5
```

Comment:
Is pyqt <5 really the requirement?  Maintaining qt 4 stuff is pretty painful.  If it is the only way, you're obviously stuck, but if you have a choice here, please let it use pyqt 5.

Reply:
Yes I am planning on moving to PyQt5. Riverbank stopped distributing binaries, and I think Debian is removing it. This is my current status but hopefully next version will be `pyqt>=5` ☺️